chapters/revisions.tex

```diff
@@ -35,7 +35,7 @@ \subsection{Main Changes in Modelica 3.6}\label{main-changes-in-modelica-3-6}
 \item Clarified lookup for choices \specpull{3307}.
 \item Constrain array assignment with implicit iterators \specpull{3306} closing \specissue{3250}.
 \item Refence connector balancing restriction \specpull{3305} closing \specissue{3260}.
-\item Correction: connect-clause was renamed to be connect-equation \specpull{3304}.
+\item Correction: connect-clause was renamed to be \lstinline!connect!-equation \specpull{3304}.
```

Comment:
Changing the revision-notes wasn't needed, but seems simpler to accept and the improve revisions in general.

Reply:
One reason for changing revision-notes is that leaving the wrong styling in this chapter makes it harder to scan the whole document for styling issues.